scripts/fledge

```diff
@@ -435,6 +435,15 @@ fledge_kill() {
 
 }
 
+fledge_debug() {
+	service="$1"
+	result=`fledge_status "silent"`
+	if [[ $result != "0" ]]; then
+		fledge_log "info" "Fledge appears not to be running, debug should be performed on a running system . Start Fledge first." "all" "pretty"
```

Comment:
You might start a new sentence at "Debug should be..." If you would rather not, I would suggest a semicolon instead of a comma. There is also a space after "running system" and before the period.